package.json

```diff
@@ -81,8 +81,8 @@
     "copy-webpack-plugin": "^11.0.0",
     "css-loader": "^6.6.0",
     "css-minimizer-webpack-plugin": "^4.0.0",
-    "cypress": "9.7.0",
-    "dotenv-webpack": "8.0.1",
+    "cypress": "^10.6.0",
+    "dotenv-webpack": "7.1.1",
```

Comment:
@miram13 - could you please make sure this dependency is up-to-date with the latest?